cmd/monaco/integrationtest/account/utils_test.go

```diff
@@ -23,49 +23,10 @@ import (
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/files"
 	"github.com/spf13/afero"
 	"github.com/stretchr/testify/require"
-	"regexp"
 	"strings"
 	"testing"
 )
 
-type monacoCmd struct {
-	cmd string
-	fs  afero.Fs
-}
-
-func monaco(cmd string) *monacoCmd {
-	cmd = regexp.MustCompile(`\s+`).ReplaceAllString(cmd, " ")
-	cmd = strings.Trim(cmd, " ")
-	cmd = strings.TrimPrefix(cmd, "monaco ")
-
-	return &monacoCmd{cmd: cmd}
-}
-func monacof(cmd string, args ...any) *monacoCmd {
-	return monaco(fmt.Sprintf(cmd, args...))
-}
-
-func (cmd *monacoCmd) withFs(fs afero.Fs) *monacoCmd {
-	cmd.fs = fs
-	return cmd
-}
-
-func (cmd *monacoCmd) run() error {
-	fs := cmd.fs
-	if fs == nil {
-		fs = afero.NewCopyOnWriteFs(afero.NewOsFs(), afero.NewMemMapFs())
-	}
-	fmt.Println(cmd)
-
-	cli := runner.BuildCli(fs)
-	cli.SetArgs(strings.Split(cmd.cmd, " "))
```

Comment:
Could we just provide the arguments directly here? Splitting on a space feels dangerous

Reply:
As this is nonproduction code and the input is static (the calling command will stay the same), I don't feel it is a threat.